lib/obfuscate_id.rb

```diff
@@ -22,9 +23,9 @@ def find(*args)
       options = args.slice!(0) || {}
       if has_obfuscated_id? && !options[:no_obfuscated_id]
         if scope.is_a?(Array)
-          scope.map! {|a| deobfuscate_id(a).to_i}
```

Comment:
Ok so I actually forked off of the latest release, `v0.2.0`. It appears that they fixed a bug with locking through `find` and added a `find` option that allows you to to skip the obfuscation. I'd be fine with taking the new changes while we're upgrading.